app.go

```diff
@@ -638,6 +638,20 @@ func (app *App) GetRoute(name string) Route {
 	return Route{}
 }
 
+// GetRoutes Get all routes, when filterUse equal to true. it will filter the routes registered by the middleware.
+func (app *App) GetRoutes(filterUse bool) []Route {
```

Comment:
I think this param should be variadic

Reply:
I've modified it.
`func (app *App) GetRoutes(filterUse bool) []Route` => `func (app *App) GetRoutes(filterUseOption ...bool) []Route`